scipy/stats/stats.py

```diff
@@ -1846,17 +1879,43 @@ def relfreq(a, numbins=10, defaultreallimits=None, weights=None):
 
     Examples
     --------
+    >>> import matplotlib.pyplot as plt
     >>> from scipy import stats
-    >>> a = np.array([1, 4, 2, 1, 3, 1])
-    >>> relfreqs, lowlim, binsize, extrapoints = stats.relfreq(a, numbins=4)
-    >>> relfreqs
+    >>> a = np.array([2, 4, 1, 2, 3, 2])
+    >>> res = stats.relfreq(a, numbins=4)
+    >>> res.frequency
     array([ 0.5       ,  0.16666667,  0.16666667,  0.16666667])
-    >>> np.sum(relfreqs)  # relative frequencies should add up to 1
+    >>> np.sum(res.frequency)  # relative frequencies should add up to 1
     0.99999999999999989
 
+    Create a normal distribution with 1000 random values
+
+    >>> rng = np.random.RandomState(seed=12345)
+    >>> samples = stats.norm.rvs(size=1000, random_state=rng)
+
+    Calculate cumulative frequencies
+
+    >>> res = stats.relfreq(samples, numbins=25)
+
+    Calculate space of values for x
+
+    >>> x = res.lowerlimit + np.linspace(0, res.binsize*res.frequency.size,
+                                         res.frequency.size)
+
+    Plot 'Histogram' and 'Cumulative histogram'
```

Comment:
This text doesn't match the plot below. You're only plotting one thing, a relative frequency histogram